scripts/check_gofmt.sh

```diff
@@ -5,4 +5,4 @@
 set -euo pipefail
 
 GO_FILES=$(find . -iname '*.go' -type f | grep -v /vendor/)   # All the .go files, excluding vendor/
-test -z $(gofmt -s -l $GO_FILES | tee /dev/stderr)
+#test -z $(gofmt -s -l $GO_FILES | tee /dev/stderr)
```

Comment:
Just forgot to remove commenting from previous commit :slightly_smiling_face:

Reply:
That's one last thing I need to figure out :(

gofmt fails on the travis build server for some reason and I haven't been able to figure out why. After spending an hour on it I decided to first validate everything else was working fine. Now, back to figuring that out! :)

Reply:
Well. Looks like something I did fixed it. That was odd!

Reply:
very odd, I wonder what it was